sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -134,6 +237,55 @@ def encode(
         device: str | None = None,
         **kwargs: Any,
     ) -> list[Tensor] | np.ndarray | Tensor | dict[str, Tensor] | list[dict[str, Tensor]]:
+        """
+        Computes sentence embeddings.
```

Comment:
```suggestion
        Computes sparse sentence embeddings.
```
Might be a bit clearer?